src/main/java/seedu/contax/storage/CsvManager.java

```diff
@@ -0,0 +1,50 @@
+package seedu.contax.storage;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.BiFunction;
+
+import seedu.contax.model.IndexedCsvFile;
+import seedu.contax.model.Model;
+
+
+public class CsvManager {
+    private Model model;
+    private BiFunction<Integer, String[], Boolean> lineAction;
+
+    /**
+     * Creates a CsvManager to import to specified {@code model}
+     */
+    public CsvManager(Model model, BiFunction<Integer, String[], Boolean> lineAction) {
+        this.model = model;
+        this.lineAction = lineAction;
+    }
+
+    /**
+     * Imports a given {@code IndexedCSvFile} to a {@code model}
+     * @param csvFile File to be imported
+     * @return CommandResult upon success
+     * @throws IOException if reading from file fails
+     */
+    // Return list of skipped lines
+    public List<Integer> importCsv(IndexedCsvFile csvFile) throws IOException {
+        BufferedReader importedCsv = new BufferedReader(new FileReader(csvFile.getFilePath()));
```

Comment:
You should close the file reader after using it.

Reply:
thanks for catching forgot to close, resolved in latest commit